lib/rules/media-query-no-invalid/index.cjs

```diff
@@ -97,34 +107,36 @@ const rule = (primary) => {
 
 					// Invalid boolean media features.
 					if (mediaQueryListParser.isMediaFeatureBoolean(node)) {
-						const name = node.getName().toLowerCase();
+						const name = node.getName();
```

Comment:
`.toLowerCase()`

Wasn't needed here because `HAS_MIN_MAX_PREFIX` is already case insensitive and this branch doesn't do any other checks on `name`